ui/src/test/java/edu/wpi/grip/ui/codegeneration/tools/CppPipelineInterfacer.java

```diff
@@ -72,7 +76,7 @@ public CppPipelineInterfacer(String libName) {
 
   @Override
   public Object getOutput(String name, GenType type) {
-    String newName = name.toLowerCase().replaceAll("_", "");
+    String newName = tMeth.getterName(name);
```

Comment:
I feel like it would make more sense to pass in the actual name instead of having to use the intermediate format that would then get transformed. 

Calling

```java
getOutput("hsvThresholdOutput", ...)
```

makes much more sense to me than

```java
getOutput("HSV_Threshold_Output", ...)
```